lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+#: messages.c:59
+msgid "Measuring reference height of calibration point"
+msgstr "Referentie hoogte van het kalibratiepunt meten"
+
+# MSG_MESH_BED_LEVELING
+#: ultralcd.cpp:5833
+msgid "Mesh Bed Leveling"
+msgstr "Mesh bed Leveling"
+
+# MSG_MMU_OK_RESUMING_POSITION c=20 r=4
+#: mmu.cpp:762
+msgid "MMU OK. Resuming position..."
+msgstr "MMU OK. Positie hervatten..."
+
+# MSG_MMU_OK_RESUMING_TEMPERATURE c=20 r=4
+#: mmu.cpp:755
+msgid "MMU OK. Resuming temperature..."
+msgstr "MMU OK. Temperatuur hervatten..."
+
+# 
+#: ultralcd.cpp:3052
+msgid "Measured skew"
+msgstr "Scheefheid"
+
+# 
+#: ultralcd.cpp:1780
+msgid "MMU fails"
+msgstr "MMU fout"
+
+# 
+#: mmu.cpp:1592
+msgid "MMU load failed     "
+msgstr "MMU laden mislukt   "
+
+# 
+#: ultralcd.cpp:1781
+msgid "MMU load fails"
+msgstr "MMU laadfout"
+
+# MSG_MMU_OK_RESUMING c=20 r=4
+#: mmu.cpp:773
+msgid "MMU OK. Resuming..."
+msgstr "MMU OK. Hervatten..."
+
+# MSG_STEALTH_MODE_OFF
+#: messages.c:90
+msgid "Mode     [Normal]"
+msgstr "Stand   [Normaal]"
+
+# MSG_SILENT_MODE_ON
+#: messages.c:89
+msgid "Mode     [silent]"
+msgstr "Stand      [stil]"
+
+# 
+#: mmu.cpp:719
+msgid "MMU needs user attention."
+msgstr "MMU heeft gebruikersaandacht nodig."
+
+# 
+#: ultralcd.cpp:1807
+msgid "MMU power fails"
+msgstr "MMU stroomstor."
+
+# MSG_STEALTH_MODE_ON
+#: messages.c:91
+msgid "Mode    [Stealth]"
+msgstr "Stand   [Stealth]"
+
+# MSG_AUTO_MODE_ON
+#: messages.c:12
+msgid "Mode [auto power]"
+msgstr "Mode[automatisch]"
+
+# MSG_SILENT_MODE_OFF
+#: messages.c:88
+msgid "Mode [high power]"
+msgstr "Stand      [hoog]"
+
+# 
+#: ultralcd.cpp:2203
+msgid "MMU2 connected"
+msgstr "MMU2 verbonden"
+
+# MSG_SELFTEST_MOTOR
+#: messages.c:83
+msgid "Motor"
+msgstr ""
+
+# MSG_MOVE_AXIS
+#: ultralcd.cpp:5719
+msgid "Move axis"
+msgstr "As verplaatsen"
+
+# MSG_MOVE_X
+#: ultralcd.cpp:4375
+msgid "Move X"
+msgstr "Verplaats X"
+
+# MSG_MOVE_Y
+#: ultralcd.cpp:4376
+msgid "Move Y"
+msgstr "Verplaats Y"
+
+# MSG_MOVE_Z
+#: ultralcd.cpp:4377
+msgid "Move Z"
+msgstr "Verplaats Z"
+
+# MSG_NO_MOVE
+#: Marlin_main.cpp:5293
+msgid "No move."
+msgstr "Geen beweging."
+
+# MSG_NO_CARD
+#: ultralcd.cpp:6878
+msgid "No SD card"
+msgstr "Geen SD kaart"
+
+# 
+#: ultralcd.cpp:3017
+msgid "N/A"
+msgstr "N/V"
+
+# MSG_NO
+#: messages.c:62
+msgid "No"
+msgstr "Nee"
+
+# MSG_SELFTEST_NOTCONNECTED
+#: ultralcd.cpp:7996
+msgid "Not connected"
+msgstr "Niet verbonden"
+
+# 
+#: util.cpp:293
+msgid "New firmware version available:"
+msgstr "Nieuwe firmware versie beschikbaar:"
+
+# MSG_SELFTEST_FAN_NO c=19
+#: messages.c:79
+msgid "Not spinning"
+msgstr "Beweegt niet"
+
+# MSG_WIZARD_V2_CAL c=20 r=8
+#: ultralcd.cpp:4957
+msgid "Now I will calibrate distance between tip of the nozzle and heatbed surface."
+msgstr "Begin met kalibratie tussen de tuit en het bed."
+
+# MSG_WIZARD_WILL_PREHEAT c=20 r=4
+#: ultralcd.cpp:5080
+msgid "Now I will preheat nozzle for PLA."
+msgstr "Opwarmen van de tuit voor PLA voor."
```

Comment:
"Voorverwarming" is probably better for preheat, e..g. "Voorverwarming van de tuit vor PLA"